Mod2/Assessments/Week2Assessment.md

```diff
@@ -0,0 +1,62 @@
+# Week 2 Assessment
+
+## Questions
```

Comment:
In past assessments we've said how many points the questions are worth. Might be nice to add that here for clarity? Maybe 10 and 10?